For Advanced/030.md

```diff
@@ -0,0 +1,37 @@
+# Automate.Net
+
+You learned how to find webpage elements and perform actions on them with the `selenium.` commands, so it’s time to see some of these functions in combat.
+
+Run this script:
+
+```G1ANT
+selenium.open chrome url google.com
+selenium.type g1ant search q by name
+delay 2
+selenium.presskey enter search btnK by name
+selenium.gettext search //*[@id="rso"]/div[1]/div/div[1]/div/div/div[1]/a[1]/div/cite by xpath result ♥url
```

Comment:
Such long path is quite risky, because it has many points and each of them might be changed any time by google. You should search by id or class; anyway we should maintain this code (and other examples from documentation) by periodic check if it still works, but without long xpath queries there's a higher change it will work for a longer period of time.

Reply:
Do you think they won't change classes when xpath is altered?